src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -210,24 +226,58 @@ The tracing integration will create a [transaction]({%- link _documentation/perf
 const activity = ApmIntegrations.Tracing.pushActivity(displayName, {
     data: {},
     op: 'react',
-    description: `<${displayName}>`,
+    description: `${displayName}`,
 });
 
 // Sometime later ...
 // When we pop the activity, the Integration will finish the span and after the timeout finish the transaction and send it to Sentry
-Integrations.ApmIntegrations.popActivity(activity);
+// Keep in mind, as long as you do not pop the activity, the transaction will be kept alive and not sent to Sentry
+ApmIntegrations.Tracing.popActivity(activity);
+```
+
+Keep in mind, if there is no active transaction you need to create one before otherwise nothing will happen.
+So given a different example where you want to create an Transaction for a user interaction on you page you need to do the following:
+
+
+```javascript
+// Let's say this function is invoked when a user clicks on the checkout button of your shop
+shopCheckout() {
+    // This will create a new Transaction for you
+    ApmIntegrations.Tracing.startIdleTransaction('shopCheckout');
```

Comment:
Why here we use `ApmIntegrations.Tracing.startIdleTransaction` instead of `hub.startSpan`?

Reply:
Because we prefer ppl using the Integration over the `startSpan`
The integration also adds spans for XHR/fetch